Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs

```diff
@@ -20,8 +21,19 @@ public string GetNews()
             List<Valute> valutes = GetTopValutes();
 
             var sb = new StringBuilder();
-            sb.AppendLine("Top - 5 exchange rates:");
-            sb.AppendLine(valutes != null ? string.Join('\n', valutes) : Errors.CanNotGetCurrencyRateFromCB);
+            // sb.AppendLine(valutes != null ? string.Join('\n', valutes) : Errors.CanNotGetCurrencyRateFromCB);
```

Comment:
What is this comment for?

Reply:
I forgot that git saves all versions and I don't need to have this :)
It was shorter way to write this function but now it returnes no correct data